navigation/navigation-common/src/commonMain/kotlin/androidx/navigation/NavDeepLink.kt

```diff
@@ -714,6 +711,10 @@ internal constructor(
     private companion object {
         private val SCHEME_PATTERN = Regex("^[a-zA-Z]+[+\\w\\-.]*:")
         private val FILL_IN_PATTERN = Regex("\\{(.+?)\\}")
+        private val SCHEME_REGEX = Regex("http[s]?://")
+        private val WILDCARD_REGEX = Regex(".*")
+        // allows for empty path arguments i.e. empty strings ""
+        private val PATH_REGEX = Regex("([^/]*?|)")
```

Comment:
this is unrelated changes but it has no behavior changes. (copy-pasted from AOSP main branch)

Reply:
https://cs.android.com/androidx/platform/frameworks/support/+/6b994dac:navigation/navigation-common/src/androidMain/kotlin/androidx/navigation/NavDeepLink.android.kt;dlc=a4a5cdc5072bfc99306eadb28097fafa3ad25237